services/core-web/src/tests/components/modalContent/__snapshots__/ReplaceDocumentModal.spec.tsx.snap

```diff
@@ -1,138 +1,3 @@
-        />
-      </svg>
-    </span>
-    <div
-      class="ant-alert-content"
-    />
-  </div>
-  <div
-    class="ant-typography margin-large--top"
-  >
-    <strong>
-      Original Document
-    </strong>
-  </div>
-  <div
-    class="ant-row ant-row-space-between padding-md--sides padding-sm--y"
-  >
-    <div
-      class="ant-col ant-col-10"
-    >
-      <span
-        class="ant-typography"
-      />
-    </div>
-    <div
-      class="ant-col"
-    >
-      <span
-        class="ant-typography"
-      />
-    </div>
-    <div
-      class="ant-col"
-    >
-      <span
-        class="ant-typography"
-      />
-    </div>
-    <div
-      class="ant-col"
-    >
-      <span
-        class="ant-typography"
-      />
-    </div>
-  </div>
-  <div
-    class="whirlpool-container"
-  >
-    <div
-      class="ant-form-item"
-    >
-      <div
-        class="ant-row ant-form-item-row"
-      >
-        <div
-          class="ant-col ant-form-item-label"
-        >
-          <label
-            class=""
-            title=""
-          />
-        </div>
-        <div
-          class="ant-col ant-form-item-control"
-        >
-          <div
-            class="ant-form-item-control-input"
-          >
-            <div
-              class="ant-form-item-control-input-content"
-            >
-              <div
-                class="filepond--wrapper"
-              >
-                <input
-                  accept="application/pdf,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/rtf,application/vnd.oasis.opendocument.text,application/vnd.oasis.opendocument.text-template,application/vnd.oasis.opendocument.text-web,application/vnd.oasis.opendocument.text-master,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,image/jpg,image/jpeg,image/png,image/jp2,image/tiff,image/heic,image/heif,application/vnd.geo+json,application/gml+xml,application/dbf,application/x-dbf,application/x-dbase,application/vnd.google-earth.kml+xml,application/vnd.google-earth.kmz,application/octet-stream,text/csv,application/xml,text/xml"
-                  multiple=""
-                  name="file"
-                  type="file"
-                />
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div
-    class="ant-modal-footer"
-  >
-    <button
-      class="ant-btn ant-btn-default full-mobile"
-      type="button"
-    >
-      <span>
-        Cancel
-      </span>
-    </button>
-    <button
-      class="ant-btn ant-btn-primary full-mobile"
-      disabled=""
-      type="button"
-    >
-      <span>
-        Replace
-      </span>
-    </button>
-  </div>
-</form>
-`;
+exports[`ReplaceDocumentModal renders properly 1`] = `undefined`;
```

Comment:
Would be good to have this snapshot render properly instead of being `undefined` if possible